### ISSUE TITLE
Add Jira reader

### ISSUE THREAD

**Issue by mb21** [jgm/pandoc#5556]
Since https://github.com/jgm/pandoc/issues/2497 we have a Jira writer to output [Jira markup](https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all).

If there's demand, we could also add a Jira markup reader to convert from it.

**Comment by phuffman** [comment]
At our company, Jira issues/tickets form the basis of our bi-weekly release notes. I'm able to access the Jira issues and convert them to Asciidoctor, but my conversion is not great and requires quite a bit of cleanup. (Jira as the basis for nicely formatted release notes would seem to be a common workflow.) A Jira reader would be terrific.

**Comment by tarleb** [comment]
@phuffman (and all others with a similar setup) please help us by sharing some more details of your current and your intended workflow. The following points would be interesting to know:

- How do you currently fetch the tickets? Do you use Jira's "Export" functionality?
- How would you like pandoc do deal with the tickets? Should pandoc read the wiki markup, or would you rather is parses XML produced by the exporter?
- Jira's exporter creates a table for each issue. Should pandoc do the same?

**Comment by phuffman** [comment]
@tarleb, thanks for your consideration.
- Our dev team works exclusively in Jira.
- The dev team lead identifies tickets of interest, populates two key fields -- a high level summary and noteworthy implementation details
- He sends me a Jira query string with the tickets of interest.
- I use a Powershell module called "JiraPS" which allows me to take the Jira query as given and iterate over the returned tickets.
- Each ticket is a Powershell object with text fields formatted as Jira wiki markup.

- I grab the few fields I need, perform a rudimentary conversion (Jira wiki to Asciidoctor), and save each ticket as an Asciidoc file.
- The result is a folder of individual Asciidoc files, one for each ticket. I create an Asciidoctor "container" document where I can sort/organize the tickets, add headers, etc.)
- As for the format of the individual Asciidoctor files ... I have a Powershell "here-string" (multi-line text string) formatted as an Asciidoc table. I do variable substitution to drop in the values.

In summary:
* I'm given a Jira query
* I use Powershell "JiraPS" to pull the tickets via api which returns Jira wiki markup.
* I do some rudimentary conversionsion on the fields I care about then drop the converted text in an Asciidoctor file, preformatted as an Asciidoctor table.
* The result is a folder of Asciidoctor files which get consolidated in an Asciidoctor container file, sorted and organized manually.

I see using the Jira wiki markup reader in the script to do the conversion, instead of the homegrown/rudimentary conversion I'm doing now.

Thanks!

**Comment by ish1301** [comment]
Jira markup to html convertor should resolve a lot issue when integrating web apps with JIRA.
